op-node/rollup/driver/state.go

```diff
@@ -331,6 +349,7 @@ func (s *state) eventLoop() {
 			if err != nil {
 				s.log.Error("Error creating new L2 block", "err", err)
 				s.metrics.DerivationErrorsTotal.Inc()
+				break // if we fail, we wait for the next block creation trigger.
```

Comment:
Noticed we were ignoring this error, and quickly rebuilding the block upon failure. Sequencing is important to not delay by as much as an exponential backoff, but on an error I think it's fair to wait till the next block-building trigger to re-attempt (it will catch up if there are no errors again). So I'm adding this `break` while we're working on the event hub anyway.

Reply:
OK, I see. Yeah I saw that there are multiple triggers that request next step, triggering block-building.